feature-utils/poly-import/src/entities/processed-data.js

```diff
@@ -0,0 +1,31 @@
+export default class ProcessedData {
```

Comment:
Pretty much all I've said above applies here. Except in this case it might actually be immutable.